xarray/testing/__init__.py

```diff
@@ -1,3 +1,4 @@
+# TODO: Add assert_isomorphic when making DataTree API public
```

Comment:
I assumed we didn't want to surface `assert_isomorphic` until everything else was public. Does that sound good to you @TomNicholas?

Reply:
I thought there was an issue collecting things we need to do to put a final bow on things, but I'm not finding it. Should we add it to #8572? or is that overkill?

Reply:
I can't find a dedicated issue for that either. Yes lets' just make an explicit list under **Expose datatree API publicly.** on #8572 (I'll do that now)